### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by EwoutH** [mesa/mesa#2201] [General]
> Good Evening !
> I want to contribute to this organization but I'm not getting how to start my work . Kindly anyone can help me get out of this.

_Originally posted by @JayaLakshmiPilla in https://github.com/projectmesa/mesa/issues/2190#issuecomment-2282778965_

**Comment by EwoutH** [comment]
Hi @JayaLakshmiPilla, that's great! There are many things that we could use help with, see all our open issues and discussions. Pick one that seems doable, and start reading up about that. Ask questions where necessary. Check out earlier work or PRs that do similar things.

Also see our open issues at Mesa-examples: https://github.com/projectmesa/mesa-examples

GitHub also has a very extensive guide on how to contribute in general: https://opensource.guide/how-to-contribute/

Tell us a little about your experience and skills, and we might to able to find a fitting issue for you to work on!

**Comment by AdeebaNizam404** [comment]
Hii I want to contribute .... But  I  don't know how to ....

**Reply by EwoutH** [reply]
Start with telling us a bit about yourself!

And you're in luck, we have written this just for you: [CONTRIBUTING.md#i-have-no-idea-where-to-start](https://github.com/projectmesa/mesa/blob/main/CONTRIBUTING.md#i-have-no-idea-where-to-start)

**Reply by AdeebaNizam404** [reply]
@EwoutH 
 Thank you so much for the warm welcome and the guidance! 😊
